### PR TITLE
Use Content Roles in Internal PR

### PR DESCRIPTION
**Pull request by kgal-pan**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:https://jira-dc.paloaltonetworks.com/browse/CIAC-10855

## Description
In continuation of https://github.com/demisto/content/pull/34932, update to use `get_content_roles` in the internal PR.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: .github/github_workflow_scripts/create_internal_pr.py [hunk 1]
```diff
@@ -10,9 +10,8 @@
 from utils import (
     get_env_var,
     timestamped_print,
-    load_json,
     get_doc_reviewer,
-    CONTENT_ROLES_PATH
```
Comment (BEAdi) [inline .github/github_workflow_scripts/create_internal_pr.py:15]: Is there any other place this is used at?

**Review by BEAdi — APPROVED** [review]
Looks good!

File: .github/github_workflow_scripts/create_internal_pr.py (hunk 1 above)
Comment (kgal-pan) [reply .github/github_workflow_scripts/create_internal_pr.py:15]: No:

image

**Review by RotemAmit — APPROVED** [review]

**Review by AradCarmi — APPROVED** [review]
Great job !

**Pull request merged**